frontend/src/pages/CreatePost/CreatePostContainer.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+
+import { submitPost } from '../../services/createPostService';
+
+const CreatePostContainer = ({ children }) => {
+  const [errorMessage, setErrorMessage] = useState('');
```

Comment:
nit: Would it be better to make the default error message value `null` instead of empty string?

Reply:
Probably. Null has a better understood meaning. Empty string is ambiguous - was there an error with an unspecified message?